src/BasicAbility/MASA.Contrib.BasicAbility.Dcc/ConfigurationAPIClient.cs

```diff
@@ -105,7 +105,7 @@ private async Task<(string Raw, ConfigurationTypes ConfigurationType)> GetRawByK
         return FormatRaw(raw);
     }
 
-    private (string Raw, ConfigurationTypes ConfigurationType) FormatRaw(string raw)
+    private (string Raw, ConfigurationTypes ConfigurationType) FormatRaw(string? raw)
```

Comment:
It is recorded here as ConfigurationAPIClient instead of ConfigurationApiClient because the full name of API is Application Programming Interface, so it is not named after a camel